src/messages/Image/Image.tsx

```diff
@@ -0,0 +1,34 @@
+import { FC, useState } from "react";
+import { MessagePasstroughProps } from "../types";
+import { MessangerImageContext } from "./context";
+import Lightbox from "./lightbox/Lightbox";
+import ImageThumb from "./ImageThumb";
+
+
+const Image: FC<MessagePasstroughProps> = props => {
+	const { url, isDownloadable = true, altText, template = 'media', config } =
```

Comment:
`config` is coming from the props (if webchat config is meant), not from the deep `message` property

Reply:
currently from the props we only have message and action. We should add it. I will do on the next PR